nni/compression/pytorch/speedup/quantization_speedup/calibrator.py

```diff
@@ -0,0 +1,51 @@
+import tensorrt as trt
+import os
+import pycuda.driver as cuda
+from urllib.request import urlretrieve
+import numpy as np
+
+
+class Calibrator(trt.IInt8Calibrator):
+    def __init__(self, training_data, cache_file, batch_size=64, algorithm=trt.CalibrationAlgoType.ENTROPY_CALIBRATION_2):
```

Comment:
Please add the doc string for this function.

Reply:
Ok, added.